pubsub/azurepubsub/azurepubsub.go

```diff
@@ -40,11 +40,11 @@ import (
 
 	"pack.ag/amqp"
 
-	"github.com/Azure/azure-amqp-common-go"
+	common "github.com/Azure/azure-amqp-common-go"
```

Comment:
goimports added these when I saved... looking through the file they seem genuinely needed - unless I'm missing something